bin/ir_resolver.ml

```diff
@@ -76,10 +76,10 @@ let key k default =
   let doc = Irmin.Private.Conf.doc k in
   let docs = Irmin.Private.Conf.docs k in
   let docv = Irmin.Private.Conf.docv k in
-  let conv = pconv (Irmin.Private.Conf.conv k) in
+  let mk = pconv (Irmin.Private.Conf.conv k) in
```

Comment:
Unless I'm mistaken this PR does not follow the `IMPORTANT` bit of the release notes. Also I think you should be able to kill your `pconv` function.